packages/version-select/util.test.ts

```diff
@@ -1,13 +1,17 @@
-import { getVersionFromPath, removeVersionFromPath } from './util'
+import {
+  getVersionFromPath,
+  removeVersionFromPath,
+  getTargetPath,
+} from './util'
 
 describe('getVersionFromPath', () => {
   it('should return the version if it is present', () => {
     {
       const path =
-        'https://waypointproject.io/docs/v10.2.4-canary.6/waypoint-hcl/variables/deploy'
+        'https://waypointproject.io/docs/v10.2.x/waypoint-hcl/variables/deploy'
```

Comment:
Simplifying our test case — we shouldn't have `v10.2.4-canary.6` as the version for the forseeable future